e2e/utils.go

```diff
@@ -127,6 +127,35 @@ func updateSecretForEncryption(c kubernetes.Interface) error {
 	return nil
 }
 
+// updateVaultTokenForEncryption is an hack to update the secrets created by rook to
+// include the vault token key.
+func updateVaultTokenForEncryption(c kubernetes.Interface) error {
+	secrets, err := c.CoreV1().Secrets(rookNamespace).Get(context.TODO(), rbdProvisionerSecretName, metav1.GetOptions{})
+	if err != nil {
+		return err
+	}
+
+	secrets.Data["vault-token"] = []byte("sample_root_token_id")
+
+	_, err = c.CoreV1().Secrets(rookNamespace).Update(context.TODO(), secrets, metav1.UpdateOptions{})
+	if err != nil {
+		return err
+	}
+
+	secrets, err = c.CoreV1().Secrets(rookNamespace).Get(context.TODO(), rbdNodePluginSecretName, metav1.GetOptions{})
+	if err != nil {
+		return err
+	}
+
+	secrets.Data["vault-token"] = []byte("sample_root_token_id")
+
+	_, err = c.CoreV1().Secrets(rookNamespace).Update(context.TODO(), secrets, metav1.UpdateOptions{})
+	if err != nil {
```

Comment:
return err we don't need to check it here

Reply:
sure